src/js/view/controls/components/chapters.mixin.js

```diff
@@ -4,28 +4,29 @@ define([
     'parsers/captions/srt',
 ], function(_, utils, srt) {
 
-    function Cue(time, text) {
-        this.time = time;
-        this.text = text;
-        this.el = document.createElement('div');
-        this.el.className = 'jw-cue jw-reset';
-    }
+    class Cue {
+
+        constructor (time, text) {
+            this.time = time;
+            this.text = text;
+            this.el = document.createElement('div');
```

Comment:
Do we want to consider passing `document` as something like `context` for these components so they can be easily unit tested?